src/table/TableWrapper.jsx

```diff
@@ -21,16 +21,28 @@ const useStyles = makeStyles({
   paginationHidden: {
     display: 'none',
   },
+  containerMobileLayout: {
+    height: 'calc(100% - 120px)',
+  },
+  paginationMobileLayout: {
+    marginTop: '-15px',
+    '& div:nth-of-type(1)': {
+      display: 'initial',
+    },
```

Comment:
What does this do?

Reply:
it is a trick that to get the first child element of the TablePagination component that is a div.